kani-driver/src/call_cargo.rs

```diff
@@ -103,8 +103,13 @@ impl KaniSession {
         let mut pkg_args: Vec<String> = vec![];
         pkg_args.extend(["--".to_string(), self.reachability_arg()]);
 
+        let packages_to_verify = packages_to_verify(&self.args, &metadata);
+        if let Err(package_error) = packages_to_verify {
```

Comment:
Why not use question mark operator?
```rust
        let packages_to_verify = packages_to_verify(&self.args, &metadata)?;
```

Reply:
Not sure what happened here, sorry about that.